internal/fsext/ls.go

```diff
@@ -70,6 +72,14 @@ var commonIgnorePatterns = sync.OnceValue(func() ignore.IgnoreParser {
 
 		// Crush
 		".crush",
+
+		"OrbStack",
+		".local",
+		".share",
+		"Library",
```

Comment:
I feel like Library,Applications,Movies,Pictures are a bit too generic and people could have these in their normal app structure.